smallfile_wrapper/trigger_smallfile.py

```diff
@@ -0,0 +1,114 @@
+                    "--yaml-input-file", self.yaml_input_file ]
+            self.logger.info('running:' + ' '.join(cmd))
+            self.logger.info('from current directory %s' % os.getcwd())
+            try:
+                process = subprocess.check_call(cmd, stderr=subprocess.STDOUT)
+            except subprocess.CalledProcessError as e:
+                self.logger.exception(e)
+                raise SmallfileWrapperException(
+                    'smallfile_cli.py non-zero process return code %d' % e.returncode)
+            self.logger.info("completed sample {} for operation {} , results in {}".format(
+                        self.sample, operation, json_output_file))
+            with open(json_output_file) as f:
+                data = json.load(f)
+                yield data, '-results'
+
+            # process response time data
+
+            elapsed_time = float(data['results']['elapsed-time'])
+            start_time = data['results']['start-time']
+            cmd = ["python", "smallfile_rsptimes_stats.py",
+                    "--time-interval", str(max(int(elapsed_time/120.0), 1)),
+                    "--start-time", str(int(start_time)),
+                    rsptime_dir ]
+            self.logger.info("process response times with: %s" % ' '.join(cmd))
+            try:
+                process = subprocess.check_call(cmd, stderr=subprocess.STDOUT)
+            except subprocess.CalledProcessError as e:
+                self.logger.exception(e)
+                raise SmallfileWrapperException(
+                    'rsptime_stats return code %d' % e.returncode)
+            self.logger.info(
+                    "response time result for operation {} in {}".format(
+                    operation, rsptime_file))
+            with open(rsptime_file) as rf:
+                lines = [ l.strip() for l in rf.readlines() ]
+                start_grabbing = False
+                for l in lines:
+                    if l.startswith('time-since-start'):
+                        start_grabbing = True
+                    elif start_grabbing:
+                        if l == '':
+                            continue
+                        flds = l.split(',')
+                        interval = {}
+                        rsptime_date = start_time + int(flds[0])
+                        rsptime_date_str = time.strftime('%Y-%m-%dT%H:%M:%S.000Z', time.gmtime(rsptime_date))
+                        interval['date'] = rsptime_date_str
+                        interval['samples'] = int(flds[2])
+                        interval['min'] = float(flds[3])
+                        interval['max'] = float(flds[4])
+                        interval['mean'] = float(flds[5])
+                        interval['50%'] = float(flds[7])
+                        interval['90%'] = float(flds[8])
+                        interval['95%'] = float(flds[9])
+                        interval['99%'] = float(flds[10])
```

Comment:
we'll want to add some additional identifiers such as the uuid clustername testuser and so on, ptal at the other benchmarks say uperf or ycsb.

Reply:
oh, so all of that goes in every document?   Is that inefficient?  Will look at fio and see how it did it.  That explains why I can't see the data in Kibana yet perhaps.

Reply:
yes that's right, and it's not inefficient. You can think of them as primary/secondary keys that are necessary to identify each entry.

Reply:
not inefficient, as they're essentially the primary/secondary keys that help us identify each record